packages/ilib-assemble/package.json

```diff
@@ -46,7 +46,7 @@
     ],
     "repository": {
         "type": "git",
-        "url": "git@github.com:iLib-js/ilib-mono.git"
+        "url": "https://github.com/iLib-js/ilib-mono.git"
```

Comment:
When the package.json only contains this type of change, I don't think it should go into the changeset.

Reply:
Extracted those meaningless changes into separate commit b39894c - this is not included in the changeset anymore.

There is a bunch of actually broken links fixed in ae2549a and this commit does have its own changeset though.